frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  type Control,
+  type FieldErrors,
+  type FieldPath,
+  type FieldValues,
+} from 'react-hook-form';
+
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useFormController } from '~/libs/hooks/hooks.js';
+
+import checkbox from './checkbox.module.scss';
+import common from './common.module.scss';
+
+const styles = { ...common, ...checkbox };
+
+type Properties<T extends FieldValues> = {
+  control: Control<T, null>;
+  errors?: FieldErrors<T>;
```

Comment:
```suggestion
  errors: FieldErrors<T>;
```
Can we make it required?

Reply:
yes, but maybe you would want a control without error handling?